Changes.md

```diff
@@ -12,6 +12,7 @@ Improvements
 - Viewer : Tool hotkeys now work as toggles for non-exclusive tools. This is currently noticeable on the Camera Tool - pressing <kbd>T</kbd> again will deactivate it.
 - ArnoldShader and ArnoldLight : Added activators and sections to the UI.
 - ImageWriter : Options are only displayed for the file format being written. If the extension is unknown, all format options are shown.
+- Main Menu : Added item to rename one or more selected nodes, with accompanying keyboard shortcut (#4286).
```

Comment:
"Main Menu" -> "Edit Menu" : I think being slightly more specific helps a little here.

Reply:
Fixed in 7c323b7d5d0057f8e52c03a402ac05a2b97f6d8d